rosdep/python.yaml

```diff
@@ -2311,6 +2400,7 @@ python-rosinstall:
 python-rosinstall-generator:
   arch: [python2-rosinstall-generator]
   fedora: [python-rosinstall_generator]
+  gentoo: [dev-util/rosinstall_generator]
```

Comment:
I mean, I for one am very strongly opposed to mixing them. We maintain ours, and we don't want the downstream people to be ahead/behind the official repo. If we are behind, please file an issue requesting that the ebuild be updated.